cli_client/python/timesketch_cli_client/commands/events.py

```diff
@@ -108,18 +98,11 @@ def annotate(ctx, timeline_id, event_id, tag, comment, output):
     required=True,
     help="Timestamp description of the event.",
 )
-@click.option(
-    "--output-format",
-    "output",
-    required=False,
-    help="Set output format (overrides global setting).",
-)
 @click.pass_context
-def add_event(ctx, message, date, attributes, timestamp_desc, output):
+def add_event(ctx, message, date, attributes, timestamp_desc):
     """Add an event to the sketch."""
     sketch = ctx.obj.sketch
-    if not output:
-        output = ctx.obj.output_format
+    output = ctx.obj.output_format
```

Comment:
When calling this function without any output definition I get a `KeyError: 'output'`. I would expect that the default output format is used if no format is defined on the cli.

Reply:
done